src/Symfony/Component/Serializer/Encoder/ChainDecoder.php

```diff
@@ -32,6 +32,8 @@ public function __construct(array $decoders = array())
 
     /**
      * {@inheritdoc}
+     * 
+     * @throws RuntimeException If no decoder is found.
```

Comment:
IMO you should always show which exception you're throwing, concrete class or not.

Theoretically one could want to extend/decorate the `ChainDecoder` specifically and in that case you may need the concrete class implementation details (if you don't then it would be better to typehint the interface instead obviously).